src/views/home/components/token-list/token-list.view.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC } from "react";
+
+import useListStyles from "src/views/home/components/token-list/token-list.styles";
+import Card from "src/views/shared/card/card.view";
+import Typography from "src/views/shared/typography/typography.view";
+import Icon from "src/views/shared/icon/icon.view";
+import Portal from "src/views/shared/portal/portal.view";
+import { Token } from "src/domain";
+
+interface TokenListProps {
+  placeholder: string;
```

Comment:
Why do we need a placeholder?

Reply:
we don't!

Reply:
Well, indeed, I remove it from `ChainList` and somehow leave it here because we will have an input. Do you think it may make sense to leave a hint like "Contract address"?

Reply:
Well, in this PR, I'm removing it. We can add it later. Done!